dashboard/config/scripts/levels/New Adaptations Project.level

```diff
@@ -0,0 +1,1553 @@
+    "hide_custom_blocks": "true",
+    "short_instructions": "Create a bird dance!",
+    "validation_code": "var astronautX = getProp({costume: \"astronaut\"}, \"x\");\r\nvar astronautY = getProp({costume: \"astronaut\"}, \"y\");\r\nif ((astronautX > 350 || astronautX < 50) && (astronautY > 350 || astronautY < 50)) {\r\n  levelSuccess(0);\r\n} else if (World.frameCount > 180) {\r\n  levelSuccess(3);\r\n}\r\n",
+    "authored_hints": "[{\"hint_class\":\"content\",\"hint_markdown\":\"Edit the behavior to solve this puzzle. Add actions into the behavior code. \",\"hint_id\":\"behavior hint\",\"hint_type\":\"general\",\"hint_path\":\"\",\"hint_video\":\"\",\"tts_url\":\"https://tts.code.org/sharon22k/180/100/aaa418bfa0e1288112962300cc121860/csc_animal_adaptations_make_behavior.mp3\"}]",
+    "long_instructions": "##Create your own behavior.\r\n\r\n**Add actions to a new behavior to model a fun tropical bird mating dance!**\r\n###Do This:\r\n- Click <code style=\"color:black; background-color:#bed7f7\">Edit</code> to open the <code style=\"color:black; background-color:#6AF36C\">dance</code> behavior\r\n- Choose 2 or 3 blocks from the **Actions** menu to put inside the <code style=\"color:black; background-color:#6AF36C\">dance</code> behavior\r\n- Click close to return to the main code\r\n- Click <code style=\"color:black; background-color:#FFBD46\">Run</code> to test your code",
+    "auto_run_setup": "DRAW_LOOP",
+    "parent_level_id": 15503,
+    "callout_json": "[\r\n {\r\n  \"localization_key\": \"behaviors_edit\",\r\n  \"callout_text\": \"Add code to this behavior.\",\r\n  \"element_id\": \"#callout\",\r\n  \"on\": \"\",\r\n  \"qtip_config\": {\r\n   \"codeStudio\": {\r\n    \"canReappear\": false,\r\n    \"dropletPaletteCategory\": \"\"\r\n   },\r\n   \"style\": {\r\n    \"classes\": \"\"\r\n   },\r\n   \"position\": {\r\n    \"my\": \"top right\",\r\n    \"at\": \"bottom right\",\r\n    \"adjust\": {\r\n     \"x\": -15,\r\n     \"y\": 0\r\n    }\r\n   }\r\n  }\r\n }\r\n]",
+    "encrypted": "false",
+    "mini_rubric": "false",
+    "encrypted_examples": "fMF3FV5MzXOakqxo8NXETKQpLfkvPHEx9araR1dB/VZWPnXbcpo2k2VUMUJb\nRGL9yvx3ji20YjjOnS1PCLEdCw==\n",
+    "validation_enabled": "false",
+    "mini_toolbox": "false",
+    "hide_pause_button": "false",
+    "standalone_app_name": "spritelab",
```

Comment:
the standalone_app_name here should be 'adaptations'

Reply:
Oh good catch! updated